values/vault-operator/vault-operator-raw.gotmpl

```diff
@@ -292,3 +320,12 @@ resources:
       - kind: ServiceAccount
         name: vault
         namespace: vault
+{{- if $v._derived.untrustedCA }}
+  - apiVersion: v1
+    kind: Secret
+    metadata:
+      name: {{ "root-ca" }}
+    type: Opaque
+    data:
+      tls.crt: {{ ($cm | get "customRootCA")| b64enc }}
```

Comment:
maybe you can also use the `$v._derived.caCert` here

Reply:
do you meant to already have it encoded in `_derived`?

Reply:
If so then it would have to be refactored in other places as well. You are welcome to contribute to this PR.

Reply:
done